eureka/S3_data_reduction/miri.py

```diff
@@ -224,9 +224,10 @@ def flag_bg(data, meta):
     return nircam.flag_bg(data, meta)
 
 
-def fit_bg(data, meta, n, isplots=False):
+def fit_bg(dataim, datamask, n, meta, isplots=False):
     '''Fit for a non-uniform background.
 
     Uses the code written for NIRCam and untested for MIRI, but likely to still work (as long as MIRI data gets rotated)
```

Comment:
While you're at it, do you want to remove the comment saying untested for MIRI, and just say:

> Uses the code written for NIRCam which works for MIRI as long as MIRI data gets rotated